<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profilo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QPushButton" name="bottone_modifica">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>ðŸ”§Modifica
  Profilo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottone_visualizza_portafoglio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>ðŸª™Visualizza
 Portafoglio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottone_visualizza_ricevute">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>ðŸ§¾Visualizza
   Ricevute</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>Torna Indietro</string>
   </property>
  </widget>
  <widget class="QLabel" name="telefono_label_to_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>137</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14; color:white</string>
   </property>
   <property name="text">
    <string>Id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cf_label_to_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>107</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="cognome_label_to_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>77</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cognome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14; color:white</string>
   </property>
   <property name="text">
    <string>Cognome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_telefono">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14; color:white</string>
   </property>
   <property name="text">
    <string>Telefono:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nome_label_to_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>47</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_label_to_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>17</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14; color:white</string>
   </property>
   <property name="text">
    <string>C. Fiscale:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14; color:white</string>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
